bin/vagrant

```diff
@@ -78,6 +78,12 @@ if argv.include?("--debug-timestamp")
   ENV["VAGRANT_LOG_TIMESTAMP"] = "1"
 end
 
+# Setting to enable/disable showing progress bars
+if argv.include?("--quiet-progress")
```

Comment:
this bit should probably change to follow new global cli options after this PR https://github.com/hashicorp/vagrant/pull/11471 gets merged. Also, rename this to `non-interactive` or something

Reply:
After discussion internally I think we landed on `--no-tty`

Reply:
Global options PR got moved to #11473 (just FYI)